testdata/smoke_config.yml

```diff
@@ -7,6 +7,12 @@ database:
 configdir: "config_test"
 client_binaries_dir: "clients"
 
+container_pools:
```

Comment:
You don't need to make this change unless you're planning on adding something to the smoke test for this

Reply:
Not planning on adding something to smoke test, but thought I would add this here so that running smoke-start-server and going to the admin page would show an example (like what the Plugins panel does).

Reply:
The purpose of this file isn't to be sample data though. Let's hold off on adding this to the smoke config until we add it to the smoke test